Docker/percep.Dockerfile

```diff
@@ -0,0 +1,156 @@
+    -D BUILD_opencv_tracking=OFF -D BUILD_opencv_xfeatures2d=OFF -D BUILD_opencv_ximgproc=OFF \
+    -D BUILD_opencv_xobjdetect=OFF -D BUILD_opencv_xphoto=OFF /opt/opencv-3.2.0/ && make -j4 && make install && ldconfig
+
+RUN  cd /usr/local && rm -rf opencv-3.2.0
+
+#Install CUDA Driver
+COPY cuda-repo-ubuntu1804_10.0.130-1_amd64.deb /usr/local
+RUN cd /usr/local && \
+    apt-get update -y && apt-get upgrade -y && apt-get update && apt-get install -y gnupg2 && \
+    dpkg -i cuda-repo-ubuntu1804_10.0.130-1_amd64.deb
+RUN apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/7fa2af80.pub
+RUN apt-get update -y
+RUN apt-get install -y cuda-10-0   
+
+#Install ZED
+RUN apt-get update -y && apt-get upgrade -y 
+RUN apt-get install -y libturbojpeg0-dev
+RUN apt-get install --no-install-recommends lsb-release wget less udev sudo apt-transport-https -y && \
+    apt-get update -y && \
+    wget -O ZED_SDK_Linux_Ubuntu18.run https://download.stereolabs.com/zedsdk/3.2/cu100/ubuntu18 && \
+    chmod +x ZED_SDK_Linux_Ubuntu18.run && \
+    ./ZED_SDK_Linux_Ubuntu18.run silent && \
+    rm ZED_SDK_Linux_Ubuntu18.run && \
+    cd /usr/local && \
+    chmod a+xwr --preserve-root --recursive zed
+
+#Install VTK
+ENV TERM xterm
+RUN echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!INSTALLING VTK" && \
+    apt-get update -y && apt-get upgrade -y && apt-get install -y keyboard-configuration libboost-all-dev curl cmake libxt-dev && \
+    apt-get install -y --no-install-recommends cmake git libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev \
+    libswscale-dev python3.6-dev python3-numpy libtbb2 libtbb-dev libjpeg-dev libpng-dev libtiff5-dev jasper \
+    libdc1394-22-dev libeigen3-dev libtheora-dev libvorbis-dev libxvidcore-dev libx264-dev sphinx-common \
+    libtbb-dev yasm libfaac-dev libopencore-amrnb-dev libopencore-amrwb-dev libopenexr-dev \
+    libgstreamer-plugins-base1.0-dev libavutil-dev libavfilter-dev libavresample-dev && \
+    curl https://vtk.org/files/release/8.2/VTK-8.2.0.tar.gz --output VTK-8.2.0.tar.gz && \
+    tar -xvzf VTK-8.2.0.tar.gz && \
+    mv VTK-8.2.0 /usr/local && \
+    cd /usr/local/VTK-8.2.0/ && \
+    mkdir VTK-Release-build && \
+    cd VTK-Release-build/ && \
+    cmake -DCMAKE_BUILD_TYPE:STRING=Release /usr/local/VTK-8.2.0/ -DVTK_USE_SYSTEM_PNG=ON && \
+    make -j1 install
+
+#Intall Eigen
+RUN echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!INSTALLING EIGEN" && \
+    cd /usr/local && \
+    wget -qO- https://gitlab.com/libeigen/eigen/-/archive/3.3.7/eigen-3.3.7.tar.gz | sudo tar xz && \
+    apt install -y libblas-dev && \
+    cd eigen-3.3.7 && sudo mkdir build && cd build && \
+    cmake .. && \
+    make install && \
+    cd ../.. && sudo rm -rf eigen-3.3.7/ && sudo rm -f eigen-3.3.7.tar.gz
+
+#Install PCL
+RUN echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!INSTALLING PCL" && \
```

Comment:
weren't you guys doing smth to remove this requirement? or am i mixing that up

Reply:
Perception is working on it. They have not committed it yet though. The changes for removing this are in #825

Reply:
its gone in GPU. Once that hits this file is going to have to change a bit, but it will actually be a significantly (10+ GB) smaller docker container.